src/components/ButtonMenu/readme.md

```diff
@@ -7,6 +7,7 @@
         <GlobalHeader className="rainbow-p-bottom_xx-large rainbow-m-bottom_xx-large">
             <ButtonGroup>
                 <ButtonMenu
+                    id="button-menu_integration-test"
```

Comment:
these ids are crazy huge, why not just `button-menu-1` remember these examples are part of the docs